hack/generate-yamls.sh

```diff
@@ -103,22 +104,13 @@ cat "${SERVING_CORE_YAML}" > "${SERVING_YAML}"
 cat "${SERVING_HPA_YAML}" >> "${SERVING_YAML}"
 cat "${SERVING_ISTIO_YAML}" >> "${SERVING_YAML}"
 
-echo "Building Monitoring & Logging"
-# Use ko to concatenate them all together.
-ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/100-namespace.yaml \
-    -f third_party/config/monitoring/logging/elasticsearch \
-    -f config/monitoring/logging/elasticsearch \
-    -f third_party/config/monitoring/metrics/prometheus \
-    -f config/monitoring/metrics/prometheus \
-    -f config/monitoring/tracing/zipkin | "${LABEL_YAML_CMD[@]}" > "${MONITORING_YAML}"
-
 # Metrics via Prometheus & Grafana
-ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/100-namespace.yaml \
+ko resolve ${KO_YAML_FLAGS} -R \
     -f third_party/config/monitoring/metrics/prometheus \
     -f config/monitoring/metrics/prometheus | "${LABEL_YAML_CMD[@]}" > "${MONITORING_METRIC_PROMETHEUS_YAML}"
 
 # Logs via ElasticSearch, Fluentd & Kibana
-ko resolve ${KO_YAML_FLAGS} -R -f config/monitoring/100-namespace.yaml \
+ko resolve ${KO_YAML_FLAGS} -R \
```

Comment:
I believe the removal of the namespace here breaks the custom installation instructions?

Reply:
@mattmoor which line in the docs did you have in mind? I agree that if people only look at the table of yaml files then they might miss the namespace yaml, but w.r.t. the instructions, they appear to push people towards using the monitor.yaml, which should be safe.

I can create a monitoring-core.yaml file (for just the NS) but it's not clear to me which docs would point to it.

Reply:
@duglin the new install docs don't reference any bundles in the main guide.  Check out: https://knative.dev/development/install/knative-with-any-k8s/#installing-the-monitoring-bundle